README.md

```diff
@@ -161,11 +161,11 @@ From source
 
     ```sh
     yum groupinstall "Development Tools"
-    yum install python python-devel libacl-devel
+    yum install python2 python2-devel libacl-devel pylibacl
     yum install fuse-python pyxattr
     yum install perl-Time-HiRes
     yum install readline-devel # optional (bup ftp)
-    yum install python-tornado # optional (bup web)
+    yum install python-tornado-doc # optional (bup web)
```

Comment:
that seems odd, why would we care about the documentation?

Reply:
If we don't care, I can change it like it was earlier.

Reply:
@jmberg  done